lib/core/src/sdk.rs

```diff
@@ -3352,8 +3361,26 @@ impl LiquidSdk {
                 self.sync_payments_with_chain_data(partial_sync).await?;
             }
         }
-        let duration_ms = Instant::now().duration_since(t0).as_millis();
-        info!("Synchronized (partial: {partial_sync}) with mempool and onchain data ({duration_ms} ms)");
+
+        let t1 = Instant::now();
+        #[cfg(all(target_family = "wasm", target_os = "unknown"))]
+        if let Err(e) = self
+            .persister
+            .backup_to_indexed_db(&self.onchain_wallet.fingerprint()?)
+            .await
+        {
+            error!("Failed to create Indexed DB backup: {e}");
+        }
```

Comment:
Also would be great to move this to the wasm crate as well. Perhaps we can register to the sync event in that crate?